src/applications/edu-benefits/0994/components/PaymentView.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
Should this be a bug with us-forms-systems that the bankAccountUI doesn't provide masking options and you had to make this ?

Reply:
I'm not sure. I've always seen it masked in the app. I reused the masking used in other areas of the app